packages/playwright-core/src/outofprocess.ts

```diff
@@ -51,7 +53,7 @@ class PlaywrightClient {
     connection.markAsRemote();
     this._transport = new IpcTransport(this._driverProcess);
     connection.onmessage = message => this._transport.send(JSON.stringify(message));
-    this._transport.onmessage = message => connection.dispatch(JSON.parse(message));
+    this._transport.onmessage = message => this._waitForNextTask(() => connection.dispatch(JSON.parse(message)));
```

Comment:
Let's move `_waitForNextTask` call into IpcTransport to be consistent with PipeTransport and WebSocketTransport.